packages/authgear-react-native/src/ui_implementation.ts

```diff
@@ -1,19 +1,23 @@
 import { openAuthorizeURL, openAuthorizeURLWithWebView } from "./nativemodule";
 
 /**
+ * OpenAuthorizationURLOptions is options for {@link UIImplementation.openAuthorizationURL}.
+ *
  * @public
  */
 export interface OpenAuthorizationURLOptions {
-  /*
-   * The URL to open.
+  /**
+   * The target authorization URL.
    */
   url: string;
-  /*
-   * The URL to detect.
+  /**
+   * The value should be a valid Redirect URI to which the response will be sent after authentication.
+   * You must also add a Redirect URI in Authgear Poral via the Redirect URI section of your Authgear Application.
    */
   redirectURI: string;
-  /*
+  /**
    * A flag to some implementations that can share cookies with the device browser.
+   * When it is set to true, cookies will be shared with the device browser.
```

Comment:
Change the description to `A flag to tell the UIImplementation that cookies should be shared with the device browser. This flag is only useful to UIImplementation that can share cookies with the device browser, such as those underlying implementations are based ASWebAuthenticationSession, or CustomTabs`.

Reply:
ok